### ISSUE TITLE
Preliminary review for design intent for my education benefits 

### ISSUE THREAD

**Issue by caw310** [department-of-veterans-affairs/vets-design-system-documentation#487]
This request is related to #26556 Design intent [DGIB - My Education Benefits] - https://github.com/department-of-veterans-affairs/va.gov-team/issues/26556

Design System Council will review to 
- Identify questions we have about DS components and 
- Application of DS in preliminary discussion
- Suggest DS components we recommend to use? 
- Suggest changes we recommend

Review page: https://app.abstract.com/share/9c202648-de36-45fb-abfc-ab8ec03df2eb?collectionId=5933c798-7ec3-4b25-b4bb-80961de86480&collectionLayerId=7b0c9738-0136-4592-b005-aea9510b9b66&present=true&preview=false&sha=d770e80f200fa441e230daf8ded28b4cf1ec7128

**Comment by caw310** [comment]
Hi @rlyons89, this will be discussed in the Design System Council meeting on Friday, July 9th.

**Comment by rlyons89** [comment]
@caw310 We are still iterating on these designs and aligning with design system components that might have been overlooked on the first draft. We will update the URL above as we can to make sure it has the latest. Does someone from our team need to be present for this meeting or can we be? Not sure what the typical protocol is

**Comment by caw310** [comment]
@rlyons89 , Yes, if you can attend the DSC meeting on Friday, that would be great. Kevin Hoffman will send you an invite. The meeting is Friday at 1pm ET.  Thanks!

**Comment by rlyons89** [comment]
@caw310 awesome, I'll look out for the invite!

**Comment by rlyons89** [comment]
@caw310 this link is the updated version of the page linked in the description above: https://app.abstract.com/projects/dc7d73cf-b7d1-4464-9725-609e27f0956f/branches/master/commits/3d4f3cb0f837962fb017278cf1582cb445a65c3f/files/000713c7-dbae-4302-be60-61559e7cc69c/layers/3F5C1E10-22A9-460D-91F3-CAE2B6964488 . We have modified the approach so that the content is displayed on the new My VA dashboard as a module.

**Comment by caw310** [comment]
@rlyons89 , that link required a login. Do you have a link that doesn't require login?

**Comment by rlyons89** [comment]
@caw310 this should work: https://share.goabstract.com/9c202648-de36-45fb-abfc-ab8ec03df2eb?collectionLayerId=e66e0a57-be91-4735-8edf-26912111b543&mode=design

**Comment by humancompanion-usds** [comment]
@rlyons89 - Has this work shipped? Was it the Card you all were looking to add to the Design System?

**Comment by humancompanion-usds** [comment]
This is lost to time.
